courseProject/pom.xml

```
@@ -40,6 +40,11 @@
 			<artifactId>lombok</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>org.springdoc</groupId>
+			<artifactId>springdoc-openapi-data-rest</artifactId>
+			<version>1.8.0</version>
```

Стоит выносить все версии зависимостей в тег properties, как это сделано с версией Джавы на 30 строке. Работа через плейсхолдеры является хорошей практикой.

Пример:
```suggestion
<properties>
    <open-api.version>1.8.0</open-api.version>
</properties>
<dependencies>
    <dependency>
	<groupId>org.springdoc</groupId>
	<artifactId>springdoc-openapi-data-rest</artifactId>
	<version>${open-api.version}</version>
    </dependency>
</dependencies>
<version>1.8.0</version>
```